### ISSUE TITLE
node modules gets loaded twice

### ISSUE THREAD

**Issue by CitySim** [NativeScript/ios-runtime#757]
The problematic modules defines a const value like this:  
`export const STORE = new StoreManager();`
If I put a `console.trace()` next to the export, I get two stacks:

```
file:///app/tns_modules/fp-core/app/orm/store/global-store.js:3:14: CONSOLE TRACE
0: anonymous(file:///app/tns_modules/fp-core/app/orm/store/global-store.js:3:14)
1: evaluate([native code])
2: moduleEvaluation()
3: (unknown)()
4: promiseReactionJob()
5: require([native code])
6: anonymous(file:///app/tns_modules/fp-core/app/orm/store/index.js:8:17)
7: evaluate([native code])
8: moduleEvaluation()
9: (unknown)()
10: promiseReactionJob()
11: require([native code])
12: anonymous(file:///app/tns_modules/fp-core/app/orm/index.js:6:17)
13: evaluate([native code])
14: moduleEvaluation()
15: (unknown)()
16: promiseReactionJob()
17: require([native code])
18: anonymous(file:///app/tns_modules/fp-core/app/index.js:10:17)
19: evaluate([native code])
20: moduleEvaluation()
21: (unknown)()
22: promiseReactionJob()
23: require([native code])
24: anonymous(file:///app/tns_modules/fp-angular/app/orm/angular-store-factory.js:3:26)
25: evaluate([native code])
26: moduleEvaluation()
27: (unknown)()
28: promiseReactionJob()
29: require([native code])
30: anonymous(file:///app/tns_modules/fp-angular/app/orm/index.js:6:17)
31: evaluate([native code])
32: moduleEvaluation()
33: (unknown)()
34: promiseReactionJob()
35: require([native code])
36: anonymous(file:///app/tns_modules/fp-angular/app/index.js:6:17)
37: evaluate([native code])
38: moduleEvaluation()
39: (unknown)()
40: promiseReactionJob()
41: require([native code])
42: anonymous(file:///app/client-storage.js:3:29)
43: evaluate([native code])
44: moduleEvaluation()
45: (unknown)()
46: promiseReactionJob()
47: require([native code])
48: anonymous(file:///app/app.module.js:11:33)
49: evaluate([native code])
50: moduleEvaluation()
51: (unknown)()
52: promiseReactionJob()
53: require([native code])
54: anonymous(file:///app/main.js:6:29)
55: evaluate([native code])
56: moduleEvaluation()
57: (unknown)()
58: promiseReactionJob()
file:///var/containers/Bundle/Application/51F9B857-ECE8-423B-BF09-11E5AC49E368/fpapp.app/app/tns_modules/fp-core/app/orm/store/global-store.js:3:14: CONSOLE TRACE
0: anonymous(file:///var/containers/Bundle/Application/51F9B857-ECE8-423B-BF09-11E5AC49E368/fpapp.app/app/tns_modules/fp-core/app/orm/store/global-store.js:3:14)
1: evaluate([native code])
2: moduleEvaluation()
3: (unknown)()
4: promiseReactionJob()
5: require([native code])
6: anonymous(file:///var/containers/Bundle/Application/51F9B857-ECE8-423B-BF09-11E5AC49E368/fpapp.app/app/tns_modules/fp-core/app/orm/store/index.js:8:17)
7: evaluate([native code])
8: moduleEvaluation()
9: (unknown)()
10: promiseReactionJob()
11: require([native code])
12: anonymous(file:///app/tns_modules/fp-core/app/orm/decorator/field.js:3:24)
13: evaluate([native code])
14: moduleEvaluation()
15: (unknown)()
16: promiseReactionJob()
17: require([native code])
18: anonymous(file:///app/tns_modules/fp-core/app/orm/decorator/index.js:6:17)
19: evaluate([native code])
20: moduleEvaluation()
21: (unknown)()
22: promiseReactionJob()
23: require([native code])
24: anonymous(file:///app/tns_modules/fp-core/app/orm/index.js:7:17)
25: evaluate([native code])
26: moduleEvaluation()
27: (unknown)()
28: promiseReactionJob()
29: require([native code])
30: anonymous(file:///app/tns_modules/fp-core/app/index.js:10:17)
31: evaluate([native code])
32: moduleEvaluation()
33: (unknown)()
34: promiseReactionJob()
35: require([native code])
36: anonymous(file:///app/tns_modules/fp-angular/app/orm/angular-store-factory.js:3:26)
37: evaluate([native code])
38: moduleEvaluation()
39: (unknown)()
40: promiseReactionJob()
41: require([native code])
42: anonymous(file:///app/tns_modules/fp-angular/app/orm/index.js:6:17)
43: evaluate([native code])
44: moduleEvaluation()
45: (unknown)()
46: promiseReactionJob()
47: require([native code])
48: anonymous(file:///app/tns_modules/fp-angular/app/index.js:6:17)
49: evaluate([native code])
50: moduleEvaluation()
51: (unknown)()
52: promiseReactionJob()
53: require([native code])
54: anonymous(file:///app/client-storage.js:3:29)
55: evaluate([native code])
56: moduleEvaluation()
57: (unknown)()
58: promiseReactionJob()
59: require([native code])
60: anonymous(file:///app/app.module.js:11:33)
61: evaluate([native code])
62: moduleEvaluation()
63: (unknown)()
64: promiseReactionJob()
65: require([native code])
66: anonymous(file:///app/main.js:6:29)
67: evaluate([native code])
68: moduleEvaluation()
69: (unknown)()
70: promiseReactionJob()
```

If only happens on a real iOS device, it works fine on the iOS Simulator.

using stable version 2.5.0

**Comment by CitySim** [comment]
if running with `tns run ios --device <id> --syncAllFiles` solved the problem

**Comment by ivanbuhov** [comment]
@CitySim We are aware of this behavior and it is fixed in `v3.0` - https://github.com/NativeScript/ios-runtime/pull/738. Let us know if the issue arises in `v3.0`.

**Comment by CitySim** [comment]
@ivanbuhov we now have updated now 3.0 and everything looks fine :)
